_security/access-control/permissions.md

```diff
@@ -69,11 +69,24 @@ Rather than individual permissions, you can often achieve your desired security
 
 These permissions are for the cluster and can't be applied granularly. For example, you either have permissions to take snapshots (`cluster:admin/snapshot/create`) or you don't. You can't have permissions to take snapshots only for certain indices.
```

Comment:
Suggestion: You can't have partial permissions. For example, you can't have permission to take snapshots of some indexes only.

Reply:
Agreed, reworded.